python/src/MarginalTransformationEvaluation_doc.i.in

```diff
@@ -6,14 +6,14 @@ Available constructors:
 
     MarginalTransformationEvaluation(*distCol, direction*)
 
-    MarginalTransformationEvaluation(*distCol, outputDistCol*)
+    MarginalTransformationEvaluation(*distCol, outputDistCol, direction*)
```

Comment:
@jschueller The documentation is slightly wrong when direction==0. You define a function through the CDF of some distributions, but this function convert a random vector into a random vector with Uniform(0,1) margins if and only if the input random vector has the given distributions as marginal distributions. If you define a marginal transformation based e.g on Exponential() distributions and you apply it to a random vector with Normal() marginal distributions then the only thing you can say is that the resulting random vector has margins taking values into [0,1].
BTW, F(X) is NOT the rank of X, even if X~F.